integration_tests/e2e/repeatCourtCaseJourney.cy.ts

```diff
@@ -680,4 +680,35 @@ context('Repeat Court Case journey', () => {
     cy.task('verifyCreateSentenceCourtAppearanceRequest').should('equal', 1)
     Page.verifyOnPageTitle(CourtCaseConfirmationPage, 'Appearance')
   })
+
```

Comment:
Ideally we could do this test in one of the above journeys - but at the moment they have been skipped with this comment
// currently failing due to the changes put in for new sentence journey. will need to be updated once all the new journey is in place.